students/20th/doeunkim/feed.js

```diff
@@ -0,0 +1,295 @@
+    if(comment.value) {
+        btn.classList.add("active");
+        btn.removeAttribute('disabled');
+    } else {
+        btn.classList.remove("active");
+        btn.setAttribute("disabled", "disabled");
+    }
+}
+
+comment.addEventListener("input", activate);
+
+
+
+//댓글 추가&좋아요&삭제
+
+const commentForm = document.querySelector(".comments form");
+
+function handlesubmit(e) {
+    e.preventDefault();
+    const currentValue = comment.value;
+    leaveaComment(currentValue);
+    comment.value = "";
+}
+
+const inputValue = document.location.search;
+const inputId = inputValue.slice(4, inputValue.indexOf('&'));
+
+function leaveaComment(text) {
+    const newComment = document.createElement("div");
+    const contents = document.querySelector(".contents");
+    newComment.className="blahblah";
+    const id = document.createElement("p");
+    id.className = "id";
+    inputId? id.innerHTML = inputId+' ' : id.innerHTML = 'unknown ';
+    const say = document.createElement("p");
+    say.innerHTML = text;
+
+    const likeComment = document.createElement("img");
+    likeComment.setAttribute('alt', 'like comment');
+    likeComment.setAttribute('src', 'img/heart.png');
+    likeComment.className = "likeComment";
+    const likeCommentRed = document.createElement("img");
+    likeCommentRed.setAttribute('alt', 'liked comment');
+    likeCommentRed.setAttribute('src', 'img/redheart.png');
+    likeCommentRed.className = "likeComment liked hide";
+
+    likeComment.addEventListener("click", ()=>{likeCommentRed.classList.toggle('hide')});
+    likeCommentRed.addEventListener("click", ()=>{likeCommentRed.classList.toggle('hide')});
+
+    const delBtn = document.createElement("img");
+    delBtn.setAttribute('alt', 'comment ellipsis');
+    delBtn.setAttribute('src', 'img/dot.png');
+    delBtn.id = "smalldot";
+    delBtn.addEventListener("click", delComment);
+
+    newComment.appendChild(id);
+    newComment.appendChild(say);
+    newComment.appendChild(delBtn);
+    newComment.appendChild(likeComment);
+    newComment.appendChild(likeCommentRed);
+    contents.appendChild(newComment);
+}
+
+function delComment() {
+    if (confirm("삭제하시겠습니까?")) {
+        this.parentElement.remove();
+    }
+}
+
+commentForm.addEventListener("submit", handlesubmit);
+
+
+
+//좋아요&취소
+
+const likes = document.querySelector("#likes");
+const red = document.querySelector("#red");
+const likeBtn = document.querySelector(".tab img");
+
+function addlike() {
+    likeBtn.classList.toggle("pop");
+    setTimeout(() => {red.style.display = "inline"}, 200);
+    likes.innerHTML = "좋아요 20개";
+}
+
+function sublike() {
+    likeBtn.classList.toggle("pop");
+    red.style.display = "none";
+    likes.innerHTML = "좋아요 19개";
+}
+
+likeBtn.addEventListener("click", addlike);
+red.addEventListener("click", sublike);
+
+
+
+//아이디 검색
+
+const typing = document.querySelector(".search input");
+const searchList = document.querySelector(".searchList");
+
+const usersToSearch = [{
```

Comment:
```suggestion
const USERS = [{
```
- 전체 유저 리스트이기 때문에 `USERS` 혹은 `USER_LIST` 등으로 간결하게 지으면 어떨까 싶습니다.
- 상수 변수 같은 경우 대문자로 사용하시면 됩니다

Reply:
감사합니다! 그런데 상수 변수가 정확히 어떤 건지 알 수 있을까요..?!